site/docs/tools/iceberg/spark.md

```diff
@@ -3,21 +3,47 @@
 !!! note    
     Detailed steps on how to set up Pyspark + Iceberg + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-iceberg-demo-nba.ipynb)
 
-To access Nessie from a spark cluster make sure the `spark.jars` spark option is set to include
-the [Spark 2](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark/{{ versions.iceberg }}/iceberg-spark-{{ versions.iceberg }}.jar)
-or [Spark 3](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark3/{{ versions.iceberg }}/iceberg-spark3-{{ versions.iceberg }}.jar) or
-[Spark 3.2](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark-runtime-{{ versions.iceberg_spark32 }}/{{ versions.iceberg }}/iceberg-spark-runtime-{{ versions.iceberg_spark32 }}-{{ versions.iceberg }}.jar) Nessie plugin jar. This fat jar
-is distributed by the Apache Iceberg project and contains all Apache Iceberg libraries required for operation, including the built-in Nessie Catalog.
+To access Nessie on Iceberg from a spark cluster make sure the `spark.jars` spark option is set to include a jar of the iceberg spark runtime, or the `spark.jars.packages` spark option is set to include a Maven coordinate of the iceberg spark runtime.
 
-In pyspark this would look like
+| | `iceberg-spark-runtime` *(required)* | `nessie-spark-extensions` *(optional)* |
+|---|:---:|:---:|
+{%- for (sparkver, scalaver, iceberg_spark_runtime, nessie_spark_extensions) in [
+  ('3.3', '2.12', artifacts.iceberg_spark_runtime_33_212, artifacts.nessie_spark_extensions_33_212),
+  ('3.3', '2.13', artifacts.iceberg_spark_runtime_33_213, artifacts.nessie_spark_extensions_33_213),
+  ('3.2', '2.12', artifacts.iceberg_spark_runtime_32_212, artifacts.nessie_spark_extensions_32_212),
+  ('3.2', '2.13', artifacts.iceberg_spark_runtime_32_213, artifacts.nessie_spark_extensions_32_213),
+  ('3.1', '2.12', artifacts.iceberg_spark_runtime_31_212, artifacts.nessie_spark_extensions_31_212),
+] %}
+| Spark **{{sparkver}}**, Scala **{{scalaver}}**: | `{{iceberg_spark_runtime.spark_jar_package}}`<br />*([All]({{iceberg_spark_runtime.all_versions_url}}), [Latest]({{iceberg_spark_runtime.jar_url}}))* | `{{nessie_spark_extensions.spark_jar_package}}`<br />*([All]({{nessie_spark_extensions.all_versions_url}}), [Latest]({{nessie_spark_extensions.jar_url}}))* |
+{%- endfor %}
+
+The `iceberg-spark-runtime` fat jars are distributed by the Apache Iceberg project and contains all Apache Iceberg libraries required for operation, including the built-in Nessie Catalog.
+
+The `nessie-spark-extensions` jars are distributed by the Nessie project and contain extensions that allow you to manage your tables with nessie's git-like syntax.
```

Comment:
```suggestion
The `nessie-spark-extensions` jars are distributed by the Nessie project and contain SQL extensions that allow you to manage your tables with nessie's git-like syntax.
```

Reply:
we can even hyperlink the "SQL extensions" to this https://projectnessie.org/tools/sql/

Reply:
Good idea!

Reply:
I've added this one with [this commit](https://github.com/projectnessie/nessie/pull/5371/commits/9edd19c6f2db677ce6671f6e292f304620c282ef).